### ISSUE TITLE
C.JSEncrypt is not a constructor

### ISSUE THREAD

**Issue by RazvanStoica0806** [travist/jsencrypt#205]
The error started to manifest in the last 2 weeks on our projects . We didn't upgrade package.json or made some changes in the modules of the application.  Before this time the applications worked just fine. 

Also, it's working on local , but on the testing environment we receive the following error :

![image](https://user-images.githubusercontent.com/48376622/109561707-38dce700-7ae6-11eb-98d1-e3d001de4f3e.png)

and the application stops loading.  

Any ideas?

**Comment by ncovercash** [comment]
this occurred for me as well when my application updated from 3.0.0-rc.1 to 3.0.1

**Comment by ncovercash** [comment]
It seems a majority of the library was restructured since 3.0.0-rc.1 https://github.com/travist/jsencrypt/compare/cc8a8c70f63e109d812c5052443610f678cc946d...master

I was able to temporarily fix this by rolling back to 3.0.0-rc.1.  For reference, my code was simply

```ts
import { JSEncrypt } from "jsencrypt";

const jse = new JSEncrypt();
...
```

**Comment by pl4yradam** [comment]
Similar to : https://github.com/travist/jsencrypt/issues/209

**Comment by travist** [comment]
Fixed in v3.1.0

**Comment by RazvanStoica0806** [comment]
This fixed it for me at the time !
Thank you !
